src/main/cpp/src/row_conversion.cu

```diff
@@ -2259,8 +2259,14 @@ std::unique_ptr<table> convert_from_rows(lists_column_view const &input,
     for (int i = 0; i < static_cast<int>(schema.size()); ++i) {
       if (schema[i].id() == type_id::STRING) {
         // stuff real string column
-        auto const null_count = string_row_offset_columns[string_idx]->null_count();
         auto string_data = string_row_offset_columns[string_idx].release()->release();
+        auto const null_count = [&] {
+          auto const &null_mask = *string_data.null_mask;
+          return null_mask.data() ?
+                     cudf::null_count(static_cast<bitmask_type const *>(null_mask.data()), 0,
+                                      num_rows) :
+                     0;
+        }();
```

Comment:
Similar to my previous comment in previous PR, put this above `string_data`:
```suggestion
auto const null_count = string_row_offset_columns[string_idx]->null_count();
```

Reply:
That is the problem that I'm trying to rectify. You'll notice that your suggestion aligns with how the code is currently. That's what we went with in #1148.

The problem is that calling `string_row_offset_columns[string_idx]->null_count()` will not automatically calculate the null-count if none was provided, after [#13372](https://github.com/rapidsai/cudf/pull/13372/files#diff-0d9ff1eeb036035ec9552be6a9f182c150233a8d68ae714de574bb8c309b1172L130) is merged.

Perhaps I have missed something?

Reply:
Oh so you meant that the column `string_row_offset_columns[string_idx]` was actually constructed without the `null_count` number provided? Then that makes sense. So please add a comment to clarify that.

Reply:
Added docs for this.